gameday/transport_test.go

```diff
@@ -0,0 +1,63 @@
+package gameday
+
+import (
+	"bytes"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/gorilla/mux"
+	log "github.com/sirupsen/logrus"
+)
+
+var logger *log.Logger
+var router *mux.Router
+
+func init() {
+	logger = log.New()
+	logger.Out = os.Stdout
+	logger.Formatter = &log.JSONFormatter{}
+}
+
+func TestHandleConfigureForm(t *testing.T) {
+
+	req, err := http.NewRequest(http.MethodPost, "/api/v1/configure/form", nil)
+	if err != nil {
+		t.Fatal("Creating 'POST /api/v1/configure/form' request failed!", err)
+	}
+
+	w := httptest.NewRecorder()
+	handler := http.HandlerFunc(HandleConfigureForm(logger))
+	handler.ServeHTTP(w, req)
+
+	logger.Info("w.Body")
+
+	if status := w.Code; status != http.StatusOK {
```

Comment:
it's totally ok to be `w.Code != http.StatusOK`. also I think we haven't any assertion for the happy case right?